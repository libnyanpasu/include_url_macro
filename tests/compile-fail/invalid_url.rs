use include_url_macro::include_url;

fn main() {
    let _content = include_url!("not_a_url");
}
