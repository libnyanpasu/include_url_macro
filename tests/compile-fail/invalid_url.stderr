error: Invalid URL: relative URL without a base
 --> tests/compile-fail/invalid_url.rs:4:20
  |
4 |     let _content = include_url!("not_a_url");
  |                    ^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `include_url` (in Nightly builds, run with -Z macro-backtrace for more info)
